README.md

```diff
@@ -4,31 +4,42 @@ Tyk provides 3 different helm charts in this repo. Please visit the respective p
 - [Tyk Hybrid](https://github.com/TykTechnologies/tyk-helm-chart/tree/master/tyk-hybrid)
 - [Tyk Headless](https://github.com/TykTechnologies/tyk-helm-chart/tree/master/tyk-headless)
 
-## Redis and MongoDB
+## Redis and MongoDB or PostgreSQL
 - Redis is required for all of the Tyk installations it must be installed in the cluster or reachable from inside K8s.
-- MongoDB is only required for the tyk-pro installation and must be installed in the cluster, or reachable from inside K8s. If you are using the mongo pumps in the tyk-headless installation you will require mongo installed for that as well.
+- MongoDB or PostgreSQL are only required for the tyk-pro installation and must be installed in the cluster, or reachable from inside K8s. If you are using the mongo or sql pumps in the tyk-headless installation you will require mongo or postgres installed for that as well.
 
-For Redis and MongoDB you can use these rather excellent charts provided by Bitnami:
+For Redis you can use these rather excellent charts provided by Bitnami:
 
 	helm repo add bitnami https://charts.bitnami.com/bitnami
 	helm repo update
 	kubectl create namespace tyk
+
+	helm install tyk-redis bitnami/redis -n tyk
+	(follow notes from the installation output to get connection details and update them in `values.yaml` file)
+
+
+For Mongo or PostgreSQL you can use these rather excellent charts provided by Bitnami:
+
 	helm install tyk-mongo bitnami/mongodb --set "replicaSet.enabled=true" -n tyk
 	(follow notes from the installation output to get connection details and update them in `values.yaml` file)
-	helm install tyk-redis bitnami/redis -n tyk
+
+	helm install tyk-postgres bitnami/postgresql --set "auth.database=tyk_analytics" -n tyk
 	(follow notes from the installation output to get connection details and update them in `values.yaml` file)
 
 *Important Note regarding MongoDB:* This helm chart enables the PodDisruptionBudget for MongoDB with an arbiter replica-count of 1.  If you intend to perform system maintenance on the node where the MongoDB pod is running and this maintenance requires for the node to be drained, this action will be prevented due the replica count being 1.  Increase the replica count in the helm chart deployment to a minimum of 2 to remedy this issue.
 
-Another option for Redis and MongoDB, if you want to get started quickly is to use our simple charts. **Please note that these provided charts must not ever be used in production and for anything but a quick start evaluation only, use external DBs or Official Helm charts for MongoDB and Redis in any other case.**
+Another option for Redis and MongoDB or PostgreSQL, if you want to get started quickly is to use our simple charts. **Please note that these provided charts must not ever be used in production and for anything but a quick start evaluation only, use external DBs or Official Helm charts for MongoDB or PostgreSQL and Redis in any other case.**
```

Comment:
I don't think we have simple charts for postgreSQL

Reply:
No we don't

Reply:
Should we remove PostgreSQL from that part then?